apps/hassio/metadata.json

```diff
@@ -0,0 +1,19 @@
+{ "id": "hassio",
+  "name": "Home Assistant API Interface",
+  "shortName":"Hassio",
+  "icon": "hassio.png",
+  "version":"0.01",
+  "description": "This app gives access to viewing Home Assistant data and sends health, compass, accelerometer, and battery information to Home Assistant as a sensor through the Home Assistant REST API.",
+  "tags": "tool,sensors",
+  "supports": ["BANGLEJS2"],
+  "dependencies": {"android":"app"},
+  "interface": "interface.html",
+  "readme": "README.md",
+  "storage": [
+    {"name":"hassio.app.js","url":"hassio.app.js"},
+    {"name":"hassio.boot.js","url":"hassio.boot.js"},
+    {"name":"hassio.img","url":"hassio.img"},
+    {"name":"hassio.json","url":"hassio.json"}
```

Comment:
For the settings, we can pop them in the `data` object, which means they won't get trashed by the app loader, for example:

https://github.com/espruino/BangleApps/blob/3f87305a2d773fd1249da95a18fabf8ca34180da/apps/worldclock/metadata.json#L17-L19